### PR TITLE
Add New Line

### PR DESCRIPTION
**Pull request by aali305**
# <Feature Title>

Tracker ID: **_#ADD LINK TO ISSUE_**

PR Branch
**_#ADD LINK TO PR BRANCH_**

Description:

### CODE REVIEW

File: new_file_2 [hunk 1]
```diff
@@ -1 +1,3 @@
 All warfare is based on deception
+
+The purpose of life is to create
```
Comment (aali305) [inline new_file_2:3]: You smart

Comment (AdrianSalondaguit) [reply new_file_2:3]: yeeeeeee

Comment (aali305) [inline new_file_2:3]: You loyal

**Review by AdrianSalondaguit — APPROVED** [review]
Aprroved!

**Pull request merged**